### DISCUSSION TITLE
well get back the old design start menu of windows 10 rtm

### DISCUSSION THREAD

**Discussion by Adorezuki** [valinet/ExplorerPatcher#2844] [Ideas]
![windows_10_start_menu](https://github.com/valinet/ExplorerPatcher/assets/68227240/ac1e47ed-e67e-46d7-b6af-3fb8346b992b)

we need the color tails back

**Comment by feherneoh** [comment]
This would be great. Manyof the Windows Store apps still specify a tile background color that they use with their otherwise transparent tile images, but with the latest Win10 versions it gets ignored. I have a relatively large custom tile collection too where I used transparent images with the background color specified in the tile XML, but now the tiles are just plain trasnparent unless the actual image has the background.

**Reply by haydengriz** [reply]
Wait you actually customized the icons  of the tiles and got the colors?

**Reply by feherneoh** [reply]
Only on older Win10 builds, I think they removed the backgrounds with 21H2?
ExplorerPatcher gets me the same features as 22H2 does, so there are no background colors here either UNLESS I edit the tile images directly to not be transparent

**Reply by Quesolover** [reply]
super late but you can with some register editing

**Comment by haydengriz** [comment]
I would love to have this arrangement, too bad Cortana wouldn’t be functional…
